src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerTagHelperCompletionService.cs

```diff
@@ -366,37 +367,5 @@ private static IReadOnlyList<TagHelperDescriptor> FilterFullyQualifiedCompletion
 
             return filteredList;
         }
-
-        private class ShortNameToFullyQualifiedComparer : IEqualityComparer<TagHelperDescriptor>
```

Comment:
📝 While updating nullability, I noticed that this same class was copy-pasted elsewhere. So, I made each location just share the same class.